src-v2/Judge/NTS.Judge/Adapters/Behinds/StartlistBehind.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Not.Application.Adapters.Behinds;
+using Not.Application.Ports.CRUD;
+using Not.Exceptions;
+using Not.TimerUtility;
+using NTS.Domain;
+using NTS.Domain.Core.Entities;
+using NTS.Domain.Core.Objects;
+using NTS.Judge.Blazor.Ports;
+using System.Diagnostics;
+using System.Timers;
+
+namespace NTS.Judge.Adapters.Behinds;
+public class StartlistBehind : ObservableBehind, IStartlistBehind
+{
+    private const double TimerTickInMilliseconds = 60000;
```

Comment:
Isn't there a better object to describe a span of time?

Furthermore - refresh on every 60 seconds?

Reply:
Unfortunately, TimeSpan.FromSeconds() cannot be assigned to a constant.

Reply:
Why does it have to be a constant?